release/release_tests.yaml

```diff
@@ -3464,6 +3464,34 @@
         cluster_env: app_config.yaml
         cluster_compute: multi_node_checkpointing_compute_config_gce.yaml
 
+- name: rllib_learner_e2e_module_loading
+  group: RLlib tests
+  working_dir: rllib_tests
+
+  frequency: nightly
+  team: rllib
+
+  cluster:
+    cluster_env: app_config.yaml
+    cluster_compute: multi_node_checkpointing_compute_config.yaml
```

Comment:
so we test multi-node only in release, right? not on CI?

Reply:
I just added something for the ci